src/core_modules/capture-ui/DateAndTimeFields/DateField/Date.component.js

```diff
@@ -1,7 +1,5 @@
 // @flow
```

Comment:
I know there is some code in this file that should have been refactored, but leaving that for another time. I removed the eslint errors and flow errors at least. This component should probably be moved to capture core (because of the DateCalendar), but leaving that to the issue @paschalidi has already started working on.

Reply:
yeah exactly. I am gonna be opening a PR soon. Kind low on the priority but I have done 80 percent of the work there